cime_config/testdefs/testmods_dirs/clm/smallville_dynurban_monthly/user_nl_clm

```diff
@@ -0,0 +1,12 @@
+do_transient_urban = .true.
+!KO The following run_zero_weight_urban setting is temporary until the HASURBAN methdology is implemented.
+run_zero_weight_urban = .true.
+
+! This file was created with the following NCL script:
+! /glade/p/cgd/tss/people/oleson/modify_surfdata/modify_smallville_with_dynurban.ncl
```

Comment:
- [ ] Thank you for your detailed comments on how the dyn urban test dataset was made. Can you please copy the ncl script into the testmods directory so that we can easily recreate it if needed (in case your directories get reorganized, etc.)?